client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -276,6 +271,22 @@ function MetadataService(api, $q, staticMetadata) {
                 });
             }
         },
+        removeSubjectTerm: function(term) {
+            var o = {},
```

Comment:
Please use a more meaningful name for variables, single letter variable names are accepted only as indexes in `for` loops.